collectionchallenge.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
maybe keep the php and html separate by using a require once?

Reply:
Added